client/payment-details/timeline/test/captured-event-strings.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+
+/**
+ * Internal dependencies
+ */
+import {
+	composeFXString,
+	composeFeeString,
+	feeBreakdown,
+	composeNetString,
+} from '../map-events';
+
+describe( 'Strings in captured events', () => {
+	beforeEach( () => {
+		jest.clearAllMocks();
+		global.wcpaySettings = {
+			zeroDecimalCurrencies: [ 'vnd' ],
+			connect: {
+				country: 'US',
+			},
+			currencyData: {
+				US: {
+					code: 'USD',
+					symbol: '$',
+					symbolPosition: 'left',
+					thousandSeparator: ',',
+					decimalSeparator: '.',
+					precision: 2,
+				},
+				FR: {
+					code: 'EUR',
+					symbol: '€',
+					symbolPosition: 'right_space',
+					thousandSeparator: ' ',
+					decimalSeparator: ',',
+					precision: 2,
+				},
+				GB: {
+					code: 'GBP',
+					symbol: '£',
+					symbolPosition: 'left',
+					thousandSeparator: ',',
+					decimalSeparator: '.',
+					precision: 2,
+				},
```

Comment:
`FR` and `GB` seem not to be used either within this PR, or https://github.com/Automattic/woocommerce-payments/pull/4212. Should we add some tests that utilize them, or remove them altogether?

Reply:
Make sense, just removed them!